openmp/tools/archer/ompt-tsan.cpp

```diff
@@ -167,8 +167,8 @@ DECLARE_TSAN_FUNCTION(AnnotateNewMemory, const char *, int,
 DECLARE_TSAN_FUNCTION(__tsan_func_entry, const void *)
 DECLARE_TSAN_FUNCTION(__tsan_func_exit)
 
-// RunningOnValgrind is used to detect absence of TSan and must intentionally be a nullptr.
-static int (*RunningOnValgrind)(void);
+// __tsan_init is used to detect absence of TSan and must intentionally be a nullptr.
+static void (*__tsan_init)(void);
```

Comment:
Since the function is never called, I suggest to declare the variable locally in `ompt_start_tool`.

Reply:
Agreed. I just pushed a second change that does that.